helm/prometheus-kafka-adapter/values.yaml

```diff
@@ -68,6 +68,7 @@ securityContext: {}
   # runAsUser: 1000
 
 service:
+  name: "{{ include \"prometheus-kafka-adapter.fullname\" . }}"
```

Comment:
change this in the dc repo instead. There's a place you can do it `env/prometheus-kafka-operator.yaml`

Reply:
I was planning to set it there, but this keeps the default behavior of the chart the exact same.

Reply:
Oh, also in our file structure, setting the service name directly in `env/prometheus-kafka-adapter.yaml` doesn't make it available to other service deployments. Hence the advantage to having it templated. It can be set to

prometheus-kafka-operator.yaml:
```
service:
  name: "{{ .Values.prometheus.kafkaAdapter.serivceName }}"
```

globalVars.yaml:
```
prometheus:
  kafkaAdapter:
    serviceName: foo-static-service-name
```

Reply:
You can also hard code it since the chart and release name won't change

```
prometheus:
  kafkaAdapter:
    serviceName: prometheus-kafka-adapter.prometheus-kafka-adapter.svc.cluster.local
```